motorlib/simResult.py

```diff
@@ -186,6 +186,8 @@ def getDesignation(self):
         imp = self.getImpulse()
         if imp < 1.25: # This is to avoid a domain error finding log(0)
             return 'N/A'
+        if imp > 83886000:
+            return 'A' + chr(int(math.log(imp / 83886000, 2)) + 65) + str(int(self.getAverageForce()))
```

Comment:
Minor thing, but this is a kind of gnarly line of code to repeat with just a different constant in it. I bet there's a way to clean it up and reduce repetition, even if the result is slightly longer. Also would be nice to write `83886000` as `1.25 * (2**26)` so it is less of a "magic number".